libsolidity/parsing/DocStringParser.cpp

```diff
@@ -52,8 +53,7 @@ string::const_iterator firstNonIdentifier(
 	if (currPos == _pos && isIdentifierStart(*currPos))
 	{
 		currPos++;
-		while (currPos != _end && isIdentifierPart(*currPos))
-			currPos++;
+		currPos = ranges::find_if_not(ranges::make_subrange(currPos, _end), [](char const& it) { return isIdentifierPart(it); });
```

Comment:
`isIdentifierPart` is already a function. Does it really need to be wrapped in a lambda?
```suggestion
		currPos = ranges::find_if_not(ranges::make_subrange(currPos, _end), isIdentifierPart);
```

Reply:
done